runtime/devnet/src/tests/local_fungibles.rs

```diff
@@ -0,0 +1,460 @@
+// Todo - errors:
```

Comment:
Feel these tests should be in the pop-api crate.

Reply:
Done, let me know if they need to be more expansive